packages/shared/src/components/Settings.tsx

```diff
@@ -135,6 +156,13 @@ export default function Settings({
           options={densities}
           value={spaciness}
           onChange={setSpaciness}
+          tooltip={
+            showDensityTooltip && {
+              content: getDensityTooltipContent(),
```

Comment:
This might cause some flickers because of evaluation? 
(not 100% sure, also not urgent) and same copy question

Reply:
I got this copy from Sab and just adjusted for mobile/tablet and layout with my thoughts based on this, but will confirm with him.
Didn't notice any flickering, just wanted to split the disabled info since it could be for different reasons

Reply:
ok so will totally remove this for mobile and tablet as confirmed with Sab, this way we won't have the function and the copy is already confirmed for density in list view